searchlib/src/vespa/searchlib/attribute/singlenumericattribute.h

```diff
@@ -41,10 +41,10 @@ class SingleValueNumericAttribute final : public B {
     }
 
 public:
-    SingleValueNumericAttribute(const vespalib::string & baseFileName);  // Only for testing
+    explicit SingleValueNumericAttribute(const vespalib::string & baseFileName);  // Only for testing
     SingleValueNumericAttribute(const vespalib::string & baseFileName, const AttributeVector::Config & c);
 
-    ~SingleValueNumericAttribute();
+    ~SingleValueNumericAttribute() final;
```

Comment:
Consider `override` instead of `final` (which is already implicit due to class marked final).

Reply:
Fixed